resources/views/user/home.blade.php

```diff
@@ -1,15 +1,23 @@
 @extends('user.layout')
```

Comment:
blade files are using tab indentation, but php files are using spaces
please unify the style